api/turing/api/ensemblers_api.go

```diff
@@ -62,12 +64,16 @@ func (c EnsemblersController) GetEnsembler(
 }
 
 func (c EnsemblersController) CreateEnsembler(
-	_ *http.Request,
+	req *http.Request,
 	vars RequestVars,
 	body interface{},
 ) *Response {
-	var errResp *Response
-	var project *mlp.Project
+	var (
+		ctx     = req.Context()
+		errResp *Response
+		project *mlp.Project
+	)
```

Comment:
Super mega nit: why did you bother with grouping the initialisation of `ctx` with the other variables into the brackets `()` here but not for the others? 😅